wp-smartcrop/wp-smartcrop.php

```diff
@@ -620,6 +607,9 @@ private function is_image_size_cropped( $size ) {
 			return true;
 		}
 		private function get_image_sizes() {
+			if ( $this->image_sizes ) {
```

Comment:
Would a strict check against null be more appropriate here? If we previously set $image_sizes to empty array, we end up re-evaluating it again. Thoughts on using `$this->image_sizes !== null` instead here?

Reply:
Changed to null check.